config/routes.rb

```diff
@@ -213,6 +213,10 @@
     resources :induction_coordinators, only: %i[index edit update], path: "induction-coordinators"
   end
 
+  namespace :finance do
+    resources :lead_providers, only: %i[index]
+  end
```

Comment:
Without a controller and a view this will either break or do something very default for Rails. Are we happy with that?

Reply:
That's correct, @plisovin is following up with another PR to fill that in